holoviews/plotting/bokeh/path.py

```diff
@@ -17,14 +17,34 @@ class PathPlot(ElementPlot):
         Whether to show legend for the plot.""")
 
     style_opts = ['color'] + line_properties
-    _plot_methods = dict(single='multi_line')
+    _plot_methods = dict(single='multi_line', batched='multi_line')
     _mapping = dict(xs='xs', ys='ys')
 
     def get_data(self, element, ranges=None, empty=False):
         xs = [] if empty else [path[:, 0] for path in element.data]
         ys = [] if empty else [path[:, 1] for path in element.data]
         return dict(xs=xs, ys=ys), self._mapping
 
+    def get_batched_data(self, element, ranges=None, empty=False):
+        data = defaultdict(list)
+        style = self.style.max_cycles(len(self.ordering))
+        for key, el in element.data.items():
+            self.overlay_dims = dict(zip(element.kdims, key))
+            eldata, elmapping = self.get_data(el, ranges, empty)
+            for k, eld in eldata.items():
+                data[k].extend(eld)
+            zorder = self.get_zorder(element, key, el)
+            val = style[zorder].get('color')
+            elmapping['color'] = 'color'
+            if isinstance(val, tuple):
+                val = rgb2hex(val)
+            data['color'] += [val for _ in range(len(eldata['xs']))]
+        if len(set(data.get('color'))) == 1:
```

Comment:
I'm a little confused by this bit. In the loop, you populate ``data`` with ``'color'`` but here you pop it off again?